### DISCUSSION TITLE
自定义表单组件包含多个表单组件时，怎么做校验

### DISCUSSION THREAD

**Discussion by Zex158** [ant-design/ant-design#36204] [Q&A]
如果自定义表单组件包含多个表单组件，如图，将其封装成自定义表单组件后，期望只在对应的表单组件下标红输入框，并且错误提示只在对应的组件下展示，而不是验证失败，全部标红。
![image](https://user-images.githubusercontent.com/31839604/175267452-a09e50dd-208f-404f-a1b7-4c5e55431d47.png)
（表现：select选中 输入框未填，select和输入框都会标红，并且错误提示不在对应的组件下）
请问有什么好的解决办法么

**Comment by wangzhengbo** [comment]
同问，想知道有没有好的处理方式
